tools/bidder-bot/notifier/selective.go

```diff
@@ -90,19 +94,25 @@ func (b *SelectiveNotifier) handleMsg(ctx context.Context, msg *notificationsapi
 	}
 
 	// Upcoming proposer slot hasn't started yet, so query block number for upcoming proposer slot - 2
-	timeoutCtx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	timeoutCtx, cancel := context.WithTimeout(ctx, 3*time.Second)
```

Comment:
Ideally, it would be best to create a constant or configuration parameter to avoid magic numbers in the code (not required though)

Reply:
Makes sense, in this specific scenario with rpc timeout the value is semi arbitrary, but agreed with values that have a lot of intention